src/jet_server.cpp

```diff
@@ -29,65 +30,101 @@ void JetServer::addJetState(const std::string& path)
         std::string globalId = value.get("Global ID", "").asString();
         ComponentPtr component = componentIdDict.get(globalId);
         auto properties = component.getAllProperties();
-        try
+        for(auto property : properties)
         {
-            for(auto property : properties)
+            std::string propertyName = property.getName();
+            if (!value.isMember(propertyName))
+            {
+                std::cout << "addJetState: skipping property " << propertyName << std::endl;
```

Comment:
Instead of using std::cout we should use in the future the logging mechanism from the openDAQ SDK.